protobuf/src/text_format/lexer/tokenizer.rs

```diff
@@ -199,6 +200,18 @@ impl<'a> Tokenizer<'a> {
         }
     }
 
+    pub fn next_symbol_expect_eq_oneof(
+        &mut self,
+        symbols: &[char],
+    ) -> TokenizerResult<char> {
+        for symbol in symbols {
+            if let Ok(()) = self.next_symbol_expect_eq(*symbol) {
+                return Ok(*symbol);
+            }
+        }
+        Err(TokenizerError::ExpectAnyChar(symbols.to_owned()))
```

Comment:
I'd initially used a static lifetime on the symbols parameter to avoid a copy in `.to_owned()` (albeit a tiny one for the current use case that only ever happens in the error case), but later opted for using a Vector instead for consistency with the existing code that returns owned values. Does that seem sane?